directory/datadog_checks/directory/data/conf.yaml.example

```diff
@@ -1,48 +1,57 @@
-## WARNING: Ensure the user account running the Agent (typically dd-agent) has read
-## access to the monitored directory and files.
-
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param directory - string - required
-    ## The directory to monitor. On windows, please make sure you escape back-slashes otherwise the YAML
-    ##  parser fails (eg. - directory: "C:\\Users\\foo\\Downloads").
+    ## The directory to monitor.
+    ##
+    ## On Windows, please make sure you escape back-slashes.
+    ## Otherwise, the YAML parser will fail (e.g "C:\\Users\\foo\\Downloads").
     #
-  - directory: <DIRECTORY_PATH>
+  - directory: <DIRECTORY>
 
-    ## @param name - string - optional - default: <DIRECTORY>
-    ## Tag metrics with specified name.
+    ## @param name - string - optional
```

Comment:
we should keep the default

Reply:
This doesn't seem to have a default in the code